### ISSUE TITLE
Bun crashes when scaffolding a vite project

### ISSUE THREAD

**Issue by Nanome203** [oven-sh/bun#14015]
### What version of Bun is running?

1.1.28+cf4e9cb69

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

Run `bun create vite@latest project-name`
Select any template

### What is the expected behavior?

A vite project gets created

### What do you see instead?

```pwsh
√ Select a framework: » React
√ Select a variant: » TypeScript

Scaffolding project in D:\project-name...
============================================================
Bun v1.1.28 (cf4e9cb6) Windows x64
Windows v.win10_fe
CPU: sse42 avx avx2 avx512
Args: "bun" "C:\Users\ME\AppData\Local\Temp\bunx-791187370-create-vite@latest\node_modules\create-vite\index.js" "project-name"
Features: jsc Bun.stdin(2) Bun.stdout transpiler_cache
Builtins: "bun:main" "node:child_process" "node:events" "node:fs" "node:path" "node:readline" "node:string_decoder" "node:url" "node:util/types"
Elapsed: 2134ms | User: 0ms | Sys: 15ms
RSS: 0.15GB | Peak: 0.15GB | Commit: 0.20GB | Faults: 35883

panic(main thread): index out of bounds: index 2, len 2
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.28/wa1cf4e9cbA6gggQ658lTm4krN07kioC___A0eNrLzEtJrVDILy1RyE9TSMovzUsptlLIBAsa6SjkpOYpGAEA4yML0g
```

### Additional information

Vite project is created normally in version 1.1.27

**Comment by Nanome203** [comment]
Could not get the stack trace from bun report

![image](https://github.com/user-attachments/assets/9e839b37-95e8-49df-ace1-169307327189)

**Comment by Nanome203** [comment]
Fixed in #14081
